src/module/actor/vehicle/sheet.ts

```diff
@@ -86,5 +88,35 @@ export class VehicleSheetPF2e extends ActorSheetPF2e<VehiclePF2e> {
 
         // get buttons
         $html.find(".crb-tag-selector").on("click", (event) => this.onTraitSelector(event));
+
+        // Change whether an effect is secret to players or not
+        $html.find(".effects-list [data-action=effect-toggle-unidentified]").on("click", async (event) => {
```

Comment:
no jqueriez

Reply:
I copied this from the PC sheet. Is this just a case of not adding *more* jQuery if it can be helped?